assets/samplesheet.csv

```diff
@@ -0,0 +1,2 @@
+patient,sample,lane,fastq_1,fastq_2
+PATIENT_ID,SAMPLE_PAIRED_END,LANE,/path/to/fastq/files/AEG588A1_S1_L002_R1_001.fastq.gz,/path/to/fastq/files/AEG588A1_S1_L002_R2_001.fastq.gz
```

Comment:
Should we add more examples of samplesheets?

Reply:
We would end up with quite a efw. We have examples for minimal and maximum columns for each step in the docs though, do you think that is sufficient

Reply:
yes, that's more than enough